### PR TITLE
Tests: rework slightly `test_cap_ri`

### PR DESCRIPTION
**Pull request by eugulixes**
* Simplify its docstring
* Make its code look like other test cases code in the module.

### CODE REVIEW

File: gits/test/terminal_test.py [hunk 1]
```diff
@@ -582,24 +582,24 @@ def test_esc_da(self):
         pass
 
     def test_cap_ri(self):
-        """The terminal should scroll down by 1 position when terminal's `top`
-        was chosen as maximum between terminal's `top` and terminal's `cur_y`.
-        """
-        
-        # Cursor at left-most position, `top` and `cur_y` equal 0.
-        s = ['a'] * self._terminal._right_most
+        """The terminal should have the possibility to scroll text down. """
```
Comment (dshil) [inline gits/test/terminal_test.py]: Scroll text down will be done only if ```_cur_y == _top_most```

Comment (eugulixes) [reply gits/test/terminal_test.py]: @dshil IMO the information should be added to the `_cap_ri` docstring. What do you think?

Comment (dshil) [reply gits/test/terminal_test.py]: @eugulixes, your are right, it's only a test. No need to duplicate the docstring.

**Review by dshil — APPROVED** [review]
LGTM

**Pull request merged**